code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -198,37 +205,76 @@ GLOBAL_LIST_INIT(sinew_recipes, list (
 	if(is_type_in_typecache(target, goliath_platable_armor_typecache))
 		var/obj/item/clothing/C = target
 		var/datum/armor/current_armor = C.armor
-		if(current_armor.getRating(MELEE) < 60)
-			C.armor = current_armor.setRating(melee_value = min(current_armor.getRating(MELEE) + 10, 60))
+		if(current_armor.getRating(MELEE) < 75)
```

Comment:
The armor types should either be completely exclusive of each other, or have a shared function that calculates armor values based on all of the armors present. Doing it piecemeal like this is a recipe for bugs.

Reply:
The goliath armor clothing enhancement feature was an original feature that lost its past armor percentages after the armor revamp, becoming in fact worse. Using the armor formula, I simply returned the old values after the revamp. The other armor types are purely for mech enhancement, similarly improving the goliath armor to the old percentage value lost after the revork. All according to the armor formula.  I'll add to the description of metal armor and dragon hide the option to add only to mech, and to goliath hide the option to enhance both mech and clothing(if it's not already there)

Translated with DeepL.com (free version)

Reply:
Change descriptions so you can sure know how to use hides and plates.